internal/cmd/verifymergeable/verifymergeable.go

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package verifymergeable
+
+import (
+	"github.com/gittuf/gittuf/internal/repository"
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	baseBranch string
+}
+
+func (o *options) AddFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(
+		&o.baseBranch,
+		"into",
+		"",
+		"base branch to merge into",
+	)
+	cmd.MarkFlagRequired("into") //nolint:errcheck
+}
+
+func (o *options) Run(cmd *cobra.Command, args []string) error {
+	repo, err := repository.LoadRepository()
+	if err != nil {
+		return err
+	}
+
+	return repo.VerifyMergeable(cmd.Context(), o.baseBranch, args[0])
```

Comment:
> @wlynch: It's a little weird that the base branch is a flag and the feature branch is an arg.
I wonder if we should change this to match git behavior?
>
> e.g. git merge main means merge main into the current branch, so gittuf verify-mergable feature == verify that the feature branch can be merged into the current branch?

isn't this similar to the command we have here? The arg is what will be merged from, similar to `git merge`. The `--into` could be made optional and assume the HEAD branch if empty?